docs/_includes/layouts/base.njk

```diff
@@ -1,114 +1,78 @@
 {% from 'back-link/macro.njk' import backLink %}
-{% from 'skip-link/macro.njk' import skipLink %}
 {% from "icon/macro.njk" import nhsappIcon %}
-{% from 'attributes.njk' import nhsukAttributes %}
-
-<!DOCTYPE html>
-<html lang="en" {% if htmlClasses %} class="{{ htmlClasses }}"{% endif %}>
-  <head>
-    <meta content="IE=edge" http-equiv="X-UA-Compatible">
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover">
-
-    <link href="/assets/css/app.css" rel="stylesheet" />
-    <script src="/nhsuk-frontend/dist/nhsuk.min.js" defer></script>
-    <script src="/assets/js/app.js" defer></script>
-
-    <title>{{ title }}</title>
-
-    <link rel="shortcut icon" href="/nhsuk-frontend/assets/favicons/favicon.ico" type="image/x-icon">
-    <link rel="apple-touch-icon" href="/nhsuk-frontend/assets/favicons/apple-touch-icon-180x180.png">
-    <link rel="mask-icon" href="/nhsuk-frontend/assets/favicons/favicon.svg" color="#005eb8">
-    <link rel="icon" sizes="192x192" href="/nhsuk-frontend/assets/favicons/favicon-192x192.png">
-    <meta name="msapplication-TileImage" content="/nhsuk-frontend/assets/favicons/mediumtile-144x144.png">
-    <meta name="msapplication-TileColor" content="#005eb8">
-    <meta name="msapplication-square70x70logo" content="/nhsuk-frontend/assets/favicons/smalltile-70x70.png">
-    <meta name="msapplication-square150x150logo" content="/nhsuk-frontend/assets/favicons/mediumtile-150x150.png">
-    <meta name="msapplication-wide310x150logo" content="/nhsuk-frontend/assets/favicons/widetile-310x150.png">
-    <meta name="msapplication-square310x310logo" content="/nhsuk-frontend/assets/favicons/largetile-310x310.png">
 
-    <meta property="og:url" content="https://design-system.nhsapp.service.nhs.uk/"/>
-    <meta property="og:site_name" content="nhs.uk"/>
-    <meta property="og:title" content="NHS App design system"/>
-    <meta property="og:description" content="Design and build accessible services for the NHS App. Use the NHS App design system to learn from the work and experiences of others."/>
-    <meta property="og:type" content="website"/>
-    <meta property="og:locale" content="en_GB"/>
-    <meta property="og:image" content="https://design-system.nhsapp.service.nhs.uk/assets/images/open-graph.png"/>
+{% block pageTitle -%}
+  {% if title %}{{ title }} - {% endif %}NHS App design system
+{%- endblock %}
 
-    {% block head %}{% endblock %}
-  </head>
-  <body>
-    <script>
-      document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');
-    </script>
-    {% block body %}
-      {% block skiplink %}
-        {{ skipLink({
-          "URL": "#maincontent",
-          "heading": "Skip to main content"
-        }) }}
-      {% endblock skiplink %}
+{% block head %}
+  <link href="/assets/css/app.css" rel="stylesheet" />
+  <script src="/assets/js/app.js" type="module"></script>
```

Comment:
Would you mind moving these to `{% block bodyEnd %}`?

We [fixed a typo](https://github.com/nhsuk/nhsuk-frontend/pull/1558/files) in [our guidance](https://github.com/nhsuk/nhsuk-frontend/blob/main/docs/installation/installing-with-npm.md#option-1-include-compiled-javascript) yesterday, sorry if it was confusing

```html
{% block bodyEnd %}
  <script type="module" src="/assets/js/app.js"></script>
  <script type="module" src="/nhsuk-frontend/dist/nhsuk/nhsuk-frontend.min.js"></script>
  <script type="module">
    import { initAll } from '/nhsuk-frontend/dist/nhsuk/nhsuk-frontend.min.js'
    initAll()
  </script>
{% endblock %}
```

```suggestion
```

<br>